assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -618,6 +618,13 @@ export default Component.extend({
           stagedId: null,
           rateLimited: false,
         });
+
+        // some markdown is cooked differently on the server-side, e.g.
+        // quotes, avatar images etc.
+        if (data.chat_message.cooked !== stagedMessage.cooked) {
+          stagedMessage.set("cooked", data.chat_message.cooked);
+        }
+
```

Comment:
What would happen if we had just replaced it all the time?

Reply:
I think that would be fine, just didn't want to have to do extra work in Ember land if we didn't have to :)